src/dmd/tokens.d

```diff
@@ -56,6 +56,7 @@ enum TOK : ubyte
     pragma_,
     typeid_,
     comment,
+    whitespace,
```

Comment:
I'd suggest putting new members at the end of the enum.

Reply:
To me it makes more sense to have `whitespace` near `comment`. I would keep it like this, unless we see problems popping up because of it.

Reply:
It's:
1. The huge diff it generates, and
2. The invalidating of all comments in the hand maintained headers (yeah, yeah, those `// 54` comments should be removed from the headers).

Reply:
Adding `whitespace` to the very end creates problems with the `__traits(getMember..)` / `__traits(allMembers..)`:
[https://auto-tester.puremagic.com/show-run.ghtml?projectid=1&runid=5130725&isPull=true](https://auto-tester.puremagic.com/show-run.ghtml?projectid=1&runid=5130725&isPull=true)
Looks like a different bug (maybe the fields beginning with `__` are not counted correctly).